site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -5,8 +5,9 @@ describe('Tests cases abut changing user pronouns', () => {
         cy.visit('/user_profile');
         cy.login('student');
 
-        cy.get('#pronouns_val').as('pronounsVal').click(); // Alias for pronouns value
-        cy.get('#user-pronouns-change').as('pronounsInput'); // Alias for pronouns input
+        //open pronouns form
+        cy.get('#pronouns_val').click();
+        cy.get('#user-pronouns-change').as('e');
```

Comment:
I think that the 'refactor cypress tests' PR that was merged before this one changed the variable that this was saved as, and I think that it would be better to keep the pronounsInput variable name, and change the place later in this file to be @pronounsInput instead of @e

Reply:
fixed! I changed it back to @pronounsInput

Reply:
> I think that the 'refactor cypress tests' PR that was merged before this one changed the variable that this was saved as, and I think that it would be better to keep the pronounsInput variable name, and change the place later in this file to be @pronounsInput instead of @e

fixed! i changed it back to @pronounsInput